doc/projects.md

```diff
@@ -25,6 +25,7 @@ limits.virtual-machines         | integer   | -                     | -
 limits.cpu                      | integer   | -                     | -                         | Maximum value for the sum of individual "limits.cpu" configs set on the instances of the project
 limits.memory                   | integer   | -                     | -                         | Maximum value for the sum of individual "limits.memory" configs set on the instances of the project
 limits.processes                | integer   | -                     | -                         | Maximum value for the sum of individual "limits.processes" configs set on the instances of the project
+restrict                        | boolean   | -                     | true                      | Block access to security-sensitive features
```

Comment:
This is supposed to be `restricted.

Also, I think we should include all the restricted keys in that first table, only keeping the separate section to clarify the behavior. We have some folks generating tools/UI from our docs by parsing those tables, so best keep structure mostly consistent between files :)

Reply:
I merged the two tables into a single one.